dhis-2/dhis-api/src/main/java/org/hisp/dhis/calendar/DateUnitPeriodTypeParser.java

```diff
@@ -204,6 +204,37 @@ else if ( DateUnitType.QUARTERLY == dateUnitType )
 
             return new DateInterval( start, end );
         }
+        else if ( DateUnitType.QUARTERLY_NOV == dateUnitType )
+        {
+            int year = Integer.parseInt( matcher.group( 1 ) );
+            int quarter = Integer.parseInt( matcher.group( 2 ) );
+
+            // valid quarters are from 1 - 4
+            if ( quarter < 1 || quarter > 4 )
+            {
+                return null;
+            }
+
+            int monthOffset = -2;
```

Comment:
Maybe we could have a method like "extractMonth()" where we can have a Javadoc explaining the calculation?
Also, giving a name for the numbers would make things easier to follow.